setup.py

```diff
@@ -444,11 +454,12 @@ def fixed_readme() -> str:
     package_dir={
         "symforce": "symforce",
         "symengine": "third_party/symenginepy/symengine",
+        "lcmtypes": "lcmtypes_build/lcmtypes",
     },
     # Override the extension builder with our cmake class
     cmdclass=cmdclass,
     # Build C++ extension module
-    ext_modules=[CMakeExtension("cc_sym")],
+    ext_modules=[CMakeExtension("cc_sym"), CMakeExtension("lcmtypes")],
```

Comment:
Does this run the extension build twice?

Reply:
From looking at the verbose output, it looks like it only gets run once (well, `CMakeBuild.run()` gets called only once, and it looks like there's only one pass of the cmake output).

I think the `ext_modules` argument is simply passed to the `build_ext` object in the `extensions` field. And our subclass `CMakeBuild` only performs one build per call to `run()`.

Reply:
Oh right, yep that makes sense